eisvogel.tex

```diff
@@ -854,15 +859,12 @@
   $endif$
   \vskip 2em
   \noindent {\Large \textsf{$for(author)$$author$$sep$, $endfor$}}
+  \vskip 1em
```

Comment:
@pfeifferj  could you enclose your addtion by 

```
$if(tutor)$
... // your addition
$endif$
```

this way the extra `\vskip 1em` would not be visible when not using the new variable `tutor` ...

Reply:
thanks for the pointer :)